systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicScalabilityUtils.java

```diff
@@ -0,0 +1,99 @@
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+
+public class KafkaTopicScalabilityUtils {
+    /**
+     * This class contains crucial methods to create, modify and check large amount of kafka topics
+     * */
+
+    private static final Logger LOGGER = LogManager.getLogger(KafkaTopicUtils.class);
+    private KafkaTopicScalabilityUtils() {}
+
+    public static void createTopicsViaK8s(ExtensionContext extensionContext, String namespaceName, String clusterName, String topicPrefix,
+                                          int numberOfTopics, int numberOfPartitions, int numberOfReplicas, int minInSyncReplicas) {
+        LOGGER.info("Creating topics via Kubernetes");
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String currentTopicName = topicPrefix + i;
+            LOGGER.debug("Creating topic {}", currentTopicName);
+
+            ResourceManager.getInstance().createResource(extensionContext, false, KafkaTopicTemplates.topic(
+                            clusterName, currentTopicName, numberOfPartitions, numberOfReplicas, minInSyncReplicas, namespaceName).build());
+        }
+    }
+
+    public static void waitForTopicStatus(String namespaceName, String topicPrefix, int numberOfTopics, Enum<?> state) {
+        List<CompletableFuture<?>> topics = new ArrayList<>();
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String currentTopic = topicPrefix + i;
+            topics.add(CompletableFuture.runAsync(() ->
+                    KafkaTopicUtils.waitForKafkaTopicStatus(namespaceName, currentTopic, state)
+            ));
+        }
+
+        CompletableFuture<Void> allTopics = CompletableFuture.allOf(topics.toArray(new CompletableFuture[0]))
+                .thenRun(() -> LOGGER.info("All topics are in correct state"));
+
+        allTopics.join();
+    }
+
+    public static void waitForTopicsNotReady(String namespaceName, String topicPrefix, int numberOfTopics) {
+        LOGGER.info("Verifying that topics are in NotReady state");
+        KafkaTopicScalabilityUtils.waitForTopicStatus(namespaceName, topicPrefix, numberOfTopics, CustomResourceStatus.NotReady);
+    }
+
+    public static void waitForTopicsReady(String namespaceName, String topicPrefix, int numberOfTopics) {
+        LOGGER.info("Verifying that topics are in Ready state");
+        KafkaTopicScalabilityUtils.waitForTopicStatus(namespaceName, topicPrefix, numberOfTopics, CustomResourceStatus.Ready);
+    }
+
+    public static void waitForTopicsContainConfig(String namespaceName, String topicPrefix, int numberOfTopics, Map<String, Object> config) {
+        LOGGER.info("Verifying that topics contain right config");
+        List<CompletableFuture<?>> topics = new ArrayList<>();
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String currentTopic = topicPrefix + i;
+            topics.add(CompletableFuture.runAsync(() -> {
+                LOGGER.info("Verifying topic " + currentTopic);
```

Comment:
Same here, I don't think that it's correct to write 1000 lines of log to INFO that you are verifying a topic